Source/Scene/GlobeDepth.js

```diff
@@ -429,7 +226,13 @@ GlobeDepth.prototype.update = function (
   var width = viewport.width;
   var height = viewport.height;
 
-  updateFramebuffers(this, context, width, height, hdr, clearGlobeDepth);
+  var textureChanged = this._colorFramebuffer.isDirty(width, height, hdr);
+  if (textureChanged) {
```

Comment:
I think this `if (textureChanged)` check can be removed, along with the `destroyResources()` calls, unless it's possible for only one of the two framebuffers to be dirty during an update. This is because neither framebuffer uses shared attachments, which means the built-in `isDirty` check in `FramebufferManager.update` should do the same thing.

We may have discussed this offline already, but I wanted to double check.